src/main/scala/com/vngrs/etl/Pipeline.scala

```diff
@@ -0,0 +1,89 @@
+package com.vngrs.etl
+
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+final class Pipeline[A](val rdd: RDD[A]) extends AnyVal {
+  def transform[B: ClassTag](transformer: Transformer[A, B]): Pipeline[B] = Pipeline(transformer(this.rdd))
```

Comment:
No need for _this_ keyword.
I don't like _this_ keyword, I think it doesn't add anything to meaning so it should be avoided.